src/main/java/com/example/books/dto/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.books.dto;
+
+import jakarta.validation.constraints.Min;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class UpdateBookRequestDto {
+    private String title;
+    private String author;
+    private String isbn;
+    @Min(0)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;

```

Comment:
Is it validation?

Reply:
Have you checked the name of the DTO? It is for an update request, all but one field can be null.